src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -403,21 +403,44 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
 
     @computed get clinicalAttributeIdToClinicalAttribute():{[clinicalAttributeId:string]:ClinicalAttribute} {
         if (this.props.store.clinicalAttributes.isComplete) {
-            return this.props.store.clinicalAttributes.result.reduce((map:{[clinicalAttributeId:string]:ClinicalAttribute}, next)=>{
+            let _map: {[clinicalAttributeId: string]: ClinicalAttribute} = this.props.store.clinicalAttributes.result.reduce((map:{[clinicalAttributeId:string]:ClinicalAttribute}, next)=>{
```

Comment:
you can shorten this as `_.keyBy(this.props.store.clinicalAttributes.result, c=>c.clinicalAttributeId)`